test/unit/routes/accessibility-statement.test.js

```diff
@@ -0,0 +1,27 @@
+import { startServer } from '../../../src/utils/start-server.js'
+import { paths } from '../../../src/routes/route-constants.js'
+import { constants as httpConstants } from 'http2'
+
+describe('#accessibilityStatement', () => {
+  describe('When accessed', () => {
```

Comment:
Can we delete the describes, `test` on it's own is 👌

https://kentcdodds.com/blog/avoid-nesting-when-youre-testing